spec/yaml/swiftnav/sbp/navigation.yaml

```diff
@@ -472,3 +471,123 @@ definitions:
+      full UTC time of the receiver. If the rover receiver knows the surveyed
+      position of the base station and has an RTK solution, this reports a
+      pseudo-absolute position solution using the base station position and the
+      rover's RTK baseline vector.
+
+      When explicit GPS to UTC time conversion parameters are not available from
+      the GPS navigation message, MSG_PVT will use the best available receiver
+      defaults. In this circumstance, UTC seconds may be incorrect by a few
+      seconds, but will be flagged accordingly.
+    fields:
+        - tow:
+            type: u32
+            units: ms
+            desc: GPS time of week rounded to the nearest millisecond
+        - year:
+            type: u16
+            units: years
+            desc: Year (UTC)
+        - month:
+            type: u8
+            units: months
+            desc: Month of the year (UTC) (ranges from to 1 to 12)
+        - day:
+            type: u8
+            units: days
+            desc: Day of the month (UTC) (ranges from 1 to 31)
+        - mins:
+            type: u8
+            units: minutes
+            desc: Minute of the hour (UTC) (ranges from 0 to 59)
+        - sec:
+            type: u8
+            units: s
+            desc: |
+              Seconds of the minute (UTC) (ranges from 0 to 60). The upper range
+              includes an the possibility of an extra leap second correction.
+        - ns:
+            type: s32
+            units: ns
+            desc: Nanosecond residual of UTC sec (ranges from -1e9 to 1e9)
+        - lat:
+            type: double
+            units: deg
+            desc: Latitude
+        - lon:
+            type: double
+            units: deg
+            desc: Longitude
+        - height:
+            type: s32
+            units: mm
+            desc: Height above WGS84 ellipsoid
+        - height_msl:
+            type: s32
+            units: mm
+            desc: Height above mean sea level (not implemented)
+        - h_accuracy:
+            type: double
+            units: mm
+            desc: |
+              Horizontal position accuracy estimate (not
+              implemented). Defaults to 0.
+        - v_accuracy:
+            type: u16
+            units: mm
+            desc: |
+              Vertical position accuracy estimate (not
+              implemented). Defaults to 0.
+        - n_sats:
+            type: u8
+            desc: Number of satellites used in solution.
+        - vel_n:
+            type: s32
+            units: mm/s
+            desc: Velocity North coordinate
+        - vel_e:
+            type: s32
+            units: mm/s
+            desc: Velocity East coordinate
+        - vel_d:
+            type: s32
+            units: mm/s
+            desc: Velocity Down coordinate
+        - pdop:
+            type: u16
+            units: 0.01
+            desc: Position Dilution of Precision
```

Comment:
Is the PDOP value of use if we already output h_accuracy and v_accuracy in millimeters (once implemented) and presumably also vel_accuracy as requested below?